lib/index.js

```diff
@@ -0,0 +1,8 @@
+var fs = require('fs')
+
+var riot = module.exports = require('./compiler')
+
+require.extensions['.tag'] = function(module, filename) {
+  var src = riot.compile(fs.readFileSync(filename, 'utf8'))
+  module._compile(src, filename)
```

Comment:
in case of a .tag file, the output will be the same as compiling with cli (`riot tag-library.tag`)